ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
+./ci/github/build_driver_container
+
+echo "Deploy KMMO..."
+make deploy KUSTOMIZE_CONFIG_DEFAULT=ci/install-ci
+
+
+echo "Wait until the KMMO Deployment is Available..."
+kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager --request-timeout=1m
+
+echo "Describe the Deployment and get its YAML if that failed..."
+kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+#        if: ${{ failure() }}
+
+echo "Check that the kmm_ci_a module is not loaded on the node..."
+          if minikube ssh -- lsmod | grep kmm_ci_a; then
+            echo "Unexpected lsmod output - the module should not be loaded"
+            exit 1
+          fi
+
+echo "Add an kmm-ci Module that contains a valid mapping..."
+          sed -e "s/KVER_CHANGEME/$(uname -r)/g" \
+            -e s/NAME_CHANGEME/kmm-ci/ \
+            -e s/KMOD_CHANGEME/kmm_ci_a/ \
+            ci/module-kmm-ci.template.yaml | tee module-kmm-ci.yaml
+          kubectl apply -f module-kmm-ci.yaml
+
+echo "Check that the module gets loaded on the node..."
+          until minikube ssh -- lsmod | grep kmm_ci_a; do
+            sleep 3
+          done
```

Comment:
What happens if it is not present? Should we add a timeout?

Reply:
It would be great to have a timeout, but I am not sure how to implement it in bash.

Reply:
Probably this would work?

https://man7.org/linux/man-pages/man1/timeout.1.html

Reply:
Looks like github has its own timeout https://github.com/kubernetes-sigs/kernel-module-management/actions/runs/3533048741/jobs/5928212666 but maybe it is better to have our own.